scripts/devops_tasks/tox_harness.py

```diff
@@ -435,6 +435,11 @@ def prep_and_run_tox(targeted_packages, parsed_args, options_array=[]):
         if parsed_args.tenvparallel:
             tox_execution_array.extend(["-p", "all"])
 
+        if parsed_args.tox_env == "apistub":
```

Comment:
I prefer not to do this if tox has an option to choose a specific arg from a list of args in {posargs} and pass to it to command. Something like {posargs:out_path} but this didn't work.

Reply:
I think this is fine. I see your difficulty.